test/test_image.py

```diff
@@ -368,6 +368,13 @@ def test_decode_jpeg_cuda(mode, img_path, scripted):
     # Some difference expected between jpeg implementations
     assert (img.float() - img_nvjpeg.cpu().float()).abs().mean() < 2
 
+@needs_cuda
+def test_decode_image_cuda_raises():
+    data = torch.randint(0, 127, size=(255,), device="cuda", dtype=torch.uint8)
+    exception_raised = True
```

Comment:
Nit: this can be removed now
```suggestion
```

Reply:
This shouldn't be a nit, since (amongst other things) it breaks the lint workflow: https://app.circleci.com/jobs/github/pytorch/vision/1851401

Reply:
https://github.com/pytorch/vision/pull/7408